XIVComboVX/GameData/PluginAddressResolver.cs

```diff
@@ -15,23 +17,17 @@ internal class PluginAddressResolver {
 	public IntPtr LastComboMove => this.ComboTimer + 0x4;
 	public string LastComboMoveAddr => this.LastComboMove.ToInt64().ToString(AddrFmtSpec);
 
-	public IntPtr GetAdjustedActionId { get; private set; } = IntPtr.Zero;
-	public string GetAdjustedActionIdAddr => this.GetAdjustedActionId.ToInt64().ToString(AddrFmtSpec);
-
 	public IntPtr IsActionIdReplaceable { get; private set; } = IntPtr.Zero;
 	public string IsActionIdReplaceableAddr => this.IsActionIdReplaceable.ToInt64().ToString(AddrFmtSpec);
 
 
-	internal void Setup() {
+	internal unsafe void Setup() {
 		try {
 			Service.Log.Information("Scanning for ComboTimer signature");
-			this.ComboTimer = Service.SigScanner.GetStaticAddressFromSig("F3 0F 11 05 ?? ?? ?? ?? 48 83 C7 08");
-
-			Service.Log.Information("Scanning for GetAdjustedActionId signature");
-			this.GetAdjustedActionId = Service.SigScanner.ScanText("E8 ?? ?? ?? ?? 89 03 8B 03");  // Client::Game::ActionManager.GetAdjustedActionId
+			this.ComboTimer = new IntPtr(&ActionManager.Instance()->Combo.Timer);
 
 			Service.Log.Information("Scanning for IsActionIdReplaceable signature");
-			this.IsActionIdReplaceable = Service.SigScanner.ScanText("E8 ?? ?? ?? ?? 84 C0 74 4C 8B D3");
+			this.IsActionIdReplaceable = Service.SigScanner.ScanText("E8 ?? ?? ?? ?? 84 C0 0F 84 ?? ?? ?? ?? C6 83 ?? ?? ?? ?? ?? 48 8B 5C 24");
```

Comment:
Thank fuck you have the sig for this. I knew it would be different after the engine changes they made, and I was _not_ looking forward to tracking it down!